android/app/src/main/res/values-v31/styles.xml

```diff
@@ -1,8 +1,5 @@
 <resources>
 
-    <!-- App Theme for Android versions 12 and above -->
-    <style name="AppTheme" parent="BaseAppTheme"/>
```

Comment:
App theme will be inherited from the one defined in `values-v26/styles.xml`, so not necessary.